MPChartLib/src/main/java/com/github/mikephil/charting/components/YAxis.java

```diff
@@ -38,6 +38,27 @@
      */
     protected boolean mDrawZeroLine = false;
 
+    /**
+     * flag indicating that auto scale min restriction should be used
+     */
+
+    private boolean mUseAutoScaleRestrictionMin = false;
+    /**
+     * flag indicating that auto scale max restriction should be used
+     */
+
+    private boolean mUseAutoScaleRestrictionMax = false;
+    /**
+     * restriction value of autoscale min
+     */
+
+    private float mAutoScaleMinRestriction = 0f;
```

Comment:
remove both of these two, `mAutoScaleMinRestriction` and `mAutoScaleMaxRestriction`